src/main/java/jenkins/plugins/logstash/LogstashInstallation.java

```diff
@@ -65,6 +67,9 @@ public static Descriptor getLogstashDescriptor() {
   public static final class Descriptor extends ToolDescriptor<LogstashInstallation> {
     public IndexerType type;
     public SyslogFormat syslogFormat;
+    @SuppressFBWarnings(
+      value="UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD",
+      justification="TODO: do we need this?")
```

Comment:
If you decide to delete it, make it transient first.

Reply:
I would prefer if this findbug issues are fixed in another pull request as this one is large enough.
I will look at the date formatter issues while waiting for script security plugin though as they appear to be more critical.

Reply:
And thank you for the quick turnaround!